model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -83,6 +83,26 @@ class LicenseInfoResolver(
                 licenseInfo.concludedLicenseInfo.concludedLicense?.also {
                     originalExpressions += ResolvedOriginalExpression(expression = it, source = LicenseSource.CONCLUDED)
                 }
+
+                licenseInfo.declaredLicenseInfo.authors.takeIf { it.isNotEmpty() && addAuthorsToCopyrights }?.also {
```

Comment:
I'd like to understand better in which situation this is required. Is it when the concluded license contains licenses which are not present in the declared licenses?

Reply:
If there is a curation, then in the Disclosure Document only the **concluded** license is displayed. The scanner stage is skipped for this particular package (option: _scanner.excludeConcluded_). So the information about the authors and copyrights (locations) comes from the curation file, and **not** from any lines in some pieces of code. What I saw in the code is is that the _declaredLicenses_ are given these dummy locations (from the curations file), while the _concludedLicenses_ don't get this information. From a logical perspective, as the source of the conclusion comes from the curation file, it would be logic to also add the authors from the curation file as "dummy" locations.

It is required when the ORT Analyzer is not able to pick up a license for a package. In a concrete case, this happened for the **SwiftPM** Analyzer and package [LicensePlist](https://github.com/mono0926/LicensePlist). As this package deals with licenses, there were a mass of false positives found by scancode, so it was decided to skip the scan step and instead write a curation, and add the author in the curation. This way, the license obligation to "include the copyright notice" could be met - if it would work.